### PR TITLE
Add the chunksize parameter

### PR DESCRIPTION
**Pull request by AndreyKostov**
Also change default chunkSize to 10MB (up from 5).

### CODE REVIEW

File: storagedriver/s3/s3.go [hunk 1]
```diff
@@ -34,7 +34,9 @@ const driverName = "s3"
 
 // minChunkSize defines the minimum multipart upload chunk size
 // S3 API requires multipart upload chunks to be at least 5MB
-const chunkSize = 5 * 1024 * 1024
+const minChunkSize = 5 * 1024 * 1024
```
Comment (stevvooe) [inline storagedriver/s3/s3.go]: Use shift notation: `5 << 20`

Comment (BrianBland) [inline storagedriver/s3/s3.go]: :+1:

File: storagedriver/s3/s3.go [hunk 2]
```diff
@@ -195,7 +209,7 @@ func New(params DriverParameters) (*Driver, error) {
 	// 	}
 	// }
 
-	return &Driver{s3obj, bucket, params.Encrypt, params.RootDirectory}, nil
+	return &Driver{s3obj, bucket, params.ChunkSize, params.Encrypt, params.RootDirectory}, nil
```
Comment (stevvooe) [inline storagedriver/s3/s3.go]: This struct should have keys called out. I'm surprised this passed the linter and vet.

**Comment by stevvooe** [comment]
LGTM! Merge after successful build.

File: storagedriver/s3/s3.go (hunk 2 above)
Comment (BrianBland) [inline storagedriver/s3/s3.go]: I feel like this explicitly goes against the vet/lint rules.

Comment (BrianBland) [inline storagedriver/s3/s3.go]: The `Create()` method isn't documented either despite being exported, so I know this file isn't being checked :(

**Comment by dmp42** [comment]
LGTM

**Pull request merged**